src/postgres/schema.sql

```diff
@@ -0,0 +1,15 @@
+CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
+
+CREATE TABLE bytecode (
+    id uuid PRIMARY KEY,
+    address text NOT NULL,
+    creation_bytecode text NOT NULL,
```

Comment:
Can we make these binary? I think text is quite a waste of space.
It has to allow for zero-bytes, though.

Reply:
Probably bitea format would be best. 

https://www.postgresql.org/docs/9.0/datatype-binary.html